index.html

```diff
@@ -13,35 +13,189 @@
+                    </div>
+                    
+                </li>
+                <li class="grid-item white"> 
+                    <div class="content">
+                        <h2 class="title">Data Dashboard Healthcare</h2>
+                        <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                        <div class="btn">
+                            <button type="button" class="grey-button">HTML</button>
+                            <button type="button" class="grey-button">Bootstrap</button>
+                            <button type="button" class="grey-button">Ruby</button>
+                        </div>
+                            <button type="button" class="orange-button">See project</button>
+                        </div>
+                        
+                    </li>
+                    <li class="grid-item white"> 
+                        <div class="content">
+                            <h2 class="title">Data Dashboard Healthcare</h2>
+                            <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                            <div class="btn">
+                                <button type="button" class="grey-button">HTML</button>
+                                <button type="button" class="grey-button">Bootstrap</button>
+                                <button type="button" class="grey-button">Ruby</button>
+                            </div>
+                                <button type="button" class="orange-button">See project</button>
+                            </div>
+                            
+                        </li>
+
+    </ul>
+
+    </main>
+
+    <!-- About Section -->
+    <section id="about">
+        <div id="intro">
+        <h2 id="about-me" class="orange-color">About me</h2>
+        <p class="text">Hey there! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+        <button type="button" id="abt-btn"  class="orange-button">Get my resume</button>
+    </div >
+    <hr>
+    <ul id="cv">
+        <li>
+            <h3 class="title"><img src="media/Rectangle 55.png" alt="icon">   Language</h3>
+            <ul class="no-style-deco">
+                <li class="list-item">JavaScript</li>
+                <li class="list-item">Ruby</li>
+                <li class="list-item">HTML</li>
+                <li class="list-item">CSS</li>
+            </ul>
+        </li>
+        <li>
+            <h3 class="title"><img src="media/Rectangle 56.png" alt="icon">   Frameworks</h3>
+            <ul class="no-style-deco">
+                <li class="list-item">React.js</li>
```

Comment:
- [ ] Kindly also in this section:

#### figma design:
![f5](https://user-images.githubusercontent.com/105105262/227730507-bffc9016-7181-4488-8c02-953cfe17c5d6.PNG)

#### your design:
![y5](https://user-images.githubusercontent.com/105105262/227730516-b9df22b3-2bf6-45c7-b739-beda15b92318.PNG)

- Here your styling is almost okay. But again, make sure to display all the technologies in the center of the page, for a better user experience and according to figma. 😃

Reply:
- [ ] It make a lot of difference to please implement what this reviewer is bringing to your aware ness to improve your design. Kindly watch the video above again for clue on how to do this :+1: